ibis/backends/tests/test_array.py

```diff
@@ -710,52 +718,6 @@ def test_array_filter_with_index(con, input, output, predicate):
     )
 
 
-@builtin_array
```

Comment:
This is just a duplicate test of the above `test_array_filter_with_index`, that's why I'm deleting it.